NameGame/NameGame/State/Reducer/GameReducer.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  GameReducer.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/21/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import ReSwift
+import UIKit
+
+enum GameAction: Action {
+    case setImages(Loading<[UIImage]>)
+    case setStatus(GameState.Status)
+    case playerAnswersCorrectly
+    case goToNextQuestion(Question)
+    case gameTerminated
+}
+
+final class GameReducer {
+    
+    class func handle(action: Action, state: AppState) -> AppState {
+        guard let action = action as? GameAction else {
```

Comment:
Add a `let gameState = state.gameState` to the guard. That way we won't continue forward if there's no game to mutate.